diesel/src/insertable.rs

```diff
@@ -267,3 +290,29 @@ where
         Ok(())
     }
 }
+
+#[derive(Debug)]
+pub struct OwnedBatchInsert<V, Tab> {
+    pub(crate) values: Vec<V>,
+    _marker: PhantomData<Tab>,
```

Comment:
Since this is owned, we don't need to defer grabbing the values. We can Just have this wrap `Vec<T>` where `T = <U as Insertable<Tab>>::Values`

Reply:
It is already done [that way](https://github.com/diesel-rs/diesel/pull/1581/files#diff-6b84e53c349d8eac5892281f4b21cc4dR238) if I understand the comment correctly.

Reply:
Why do we need this field, then?